README.md

```diff
@@ -1 +1,102 @@
-# deployment-deploy-services
\ No newline at end of file
+# How to deploy dalite-ng on OpenStack server
+ 
+This repository contains ansible deployment scripts allowing one to deploy dalite-ng to OpenStack server.
+
+
+Create a OpenStack environment
+------------------------------
+
+1. Create a root volume that has 40GB size. It should be based on Ubuntu 14.04 image. 
+   We suggest you use vanilla image from [ubuntu page](https://cloud-images.ubuntu.com/). 
```

Comment:
@omarkhan Uploading is covered in a separate document, which since there are more than one deployment repository is here: https://github.com/open-craft/public-doc/pull/1 .